components/form-input/src/FormInput.js

```diff
@@ -0,0 +1,200 @@
+/** @jsx jsx */
```

Comment:
Please be careful not to bring in other components that are not relevant to this pr.

Reply:
will remove them 👍

Reply:
I'll remove it for now so we can merge `core-styling` into `gel-develop`. But will have to hook this back up once we merge `form` into `gel-develop`.